### PR TITLE
Update code of the reactive routes guide 

### PR DESCRIPTION
**Pull request by cescoffier**
The HTTP method enum is now provided by @Route.HttpMethod instead of Vert.x.

### CODE REVIEW

**Review by mkouba — APPROVED** [review]

File: docs/src/main/asciidoc/reactive-routes.adoc [hunk 1]
```diff
@@ -52,27 +52,26 @@ import io.vertx.ext.web.RoutingContext;
 
 import javax.enterprise.context.ApplicationScoped;
 
-@ApplicationScoped <1>
+@ApplicationScoped // <1>
```
Comment (cuspymd) [inline docs/src/main/asciidoc/reactive-routes.adoc:55]: The number seems be displayed well, why did you add "//"?

**Review by gsmet — APPROVED** [review]

File: docs/src/main/asciidoc/reactive-routes.adoc (hunk 1 above)
Comment (cescoffier) [reply docs/src/main/asciidoc/reactive-routes.adoc:55]: Hum, it didn't when I generated locally (the copy and paste was messed up). 
I will check on the website.
